src/components/structures/MatrixChat.tsx

```diff
@@ -1206,6 +1208,30 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
     private async onLoggedIn() {
         ThemeController.isLogin = false;
         this.themeWatcher.recheck();
+        StorageManager.tryPersistStorage();
+
+        if (SettingsStore.isLevelSupported(SettingLevel.ACCOUNT) &&
+            MatrixClientPeg.currentUserIsJustRegistered() &&
+            SettingsStore.getValue("ftueUseCaseSelection") === null) {
```

Comment:
I don't think the setting level check is needed here, and the formatting makes the condition bleed into the content:

```suggestion
        if (
            MatrixClientPeg.currentUserIsJustRegistered() &&
            SettingsStore.getValue("ftueUseCaseSelection") === null
        ) {
```